backend/src/ml_space_lambda/emr/lambda_functions.py

```diff
@@ -158,7 +138,7 @@ def create(event, context):
                 },
             },
         ],
-        "Ec2KeyName": ec2_key_name,
+        "Ec2KeyName": "",
```

Comment:
Why is this always `""` now? How does using the dynamic config mean this is no longer variable?

Reply:
This will now be defined by the new `EMR_EC2_SSH_KEY` deployment variable. 

When we generate our config object we'll also see if the customer has an existing `cluster-config.json` file, and if they do we'll try to use the `ec2-key` value from that file as the default value for EMR_EC2_SSH_KEY.